samples/dnn/mobilenet_ssd_python.py

```diff
@@ -99,7 +99,7 @@
         y2 = y1 + cropSize[1]
         x1 = (cols - cropSize[0]) / 2
         x2 = x1 + cropSize[0]
-        frame = frame[y1:y2, x1:x2]
+        frame = frame[int(y1):int(y2), int(x1):int(x2)] #for Python 3 compatibility (2/2=1.0)
```

Comment:
I think `y1 = int((rows - cropSize[1]) / 2)`, `x1 = int((cols - cropSize[0]) / 2)` is even better.